test/text_shaping/test_text_shaping.py

```diff
@@ -0,0 +1,144 @@
+from pathlib import Path
+
+from fpdf import FPDF
+from test.conftest import assert_pdf_equal
+
+HERE = Path(__file__).resolve().parent
+FONTS_DIR = HERE.parent / "fonts"
+
+# Tests to be included:
+# char spacing, word spacing(justified text)
```

Comment:
Same question: should this be implemented before merging this PR?

Reply:
"test_multi_cell_markdown_with_shaping" and "test_features" are covering this